tests/unit/StaffNotation.spec.ts

```diff
@@ -26,7 +26,88 @@ document.body.setAttribute('data-app', 'true')
 
 // Prevent the warning "[Vuetify] Unable to locate target #staff-notation"
 document.body.setAttribute('id', 'staff-notation')
+describe('StaffNotation - Put Back On', () => {
+  const { assign } = window.location
+
+  beforeAll(() => {
+    // mock the window.location.assign function
+    delete window.location
+    window.location = { assign: vi.fn() } as any
+
+    const configuration = {
+      VUE_APP_BUSINESS_CREATE_URL: 'https://create.url/',
+      VUE_APP_BUSINESS_EDIT_URL: 'https://edit.url/'
+    }
+
+    // set configurations
+    configurationStore.setConfiguration(configuration)
+  })
+
+  afterAll(() => {
+    window.location.assign = assign
+  })
+
+  beforeEach(() => {
+    // allow all filings referenced in this component
+    // some of these are normally mutually exclusive, but that's OK for testing
+    businessStore.setAllowedActions({
+      digitalBusinessCard: false,
+      filing: {
+        filingTypes: [
+          { name: FilingTypes.REGISTRARS_NOTATION },
+          { name: FilingTypes.REGISTRARS_ORDER },
+          { name: FilingTypes.COURT_ORDER },
+          { name: FilingTypes.CONVERSION },
+          { name: FilingTypes.DISSOLUTION }, // FUTURE: add dissolution type
+          { name: FilingTypes.RESTORATION, type: FilingSubTypes.FULL_RESTORATION },
+          { name: FilingTypes.PUT_BACK_ON },
+          { name: FilingTypes.ADMIN_FREEZE },
+          { name: FilingTypes.CONSENT_CONTINUATION_OUT },
+          { name: FilingTypes.RESTORATION, type: FilingSubTypes.LIMITED_RESTORATION_EXTENSION },
+          { name: FilingTypes.RESTORATION, type: FilingSubTypes.LIMITED_RESTORATION_TO_FULL }
```

Comment:
Since you're only testing Put Back On in this test suite (describe block), please delete the allowed filing types that you don't care about.